lib/CorresK_Method.thy

```diff
@@ -4,11 +4,11 @@
  * SPDX-License-Identifier: BSD-2-Clause
  *)
 
-theory Corres_Method
+theory CorresK_Method
```

Comment:
We might want to write some kind of a tombstone for why CorresK exists, when people ask why there is a Corres_Method and a CorresK_Method.

Reply:
It's currently still used, so maybe a bit early for tombstone, but you're right that some kind of comment is in order.